src/planscape/users/reused_password_validator.py

```diff
@@ -0,0 +1,7 @@
+from password_policies.password_validation import ReusedPasswordValidator
+
+
+class CustomReusedPasswordValidator(ReusedPasswordValidator):
+    # This is just replacing the Chinese text from the password_policies library
+    def get_help_text(self):
```

Comment:
There's an upcoming ticket to replace this entire python library, so this felt like the best compromise between overcoming its flaws and making the password validation usable.